worker/snapshot_test.go

```
@@ -99,17 +104,25 @@ func TestSnapshot(t *testing.T) {
 		})
 		require.NoError(t, err)
 	}
+	testutil.UpdateGQLSchema(t, testutil.SockAddrHttp, testSchema)
 	_ = waitForSnapshot(t, snapshotTs)
```

are we ignoring the error here?

---

no @mangalaman93